backend/src/main/java/corea/member/domain/Profile.java

```diff
@@ -0,0 +1,37 @@
+package corea.member.domain;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+
+@Entity
+@AllArgsConstructor
+@Getter
+public class Profile {
+
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    private Long id;
+
+    private long feedbackCount;
+
+    private long receiveCount;
+
+    private long deliverCount;
+
+    private float averageRating;
+
+    private float attitude;
+
+    public Profile() {
```

Comment:
`@NoArgsConstructor`를 사용하지 않으신 건 잔디 온도(36.5f) 때문이겠죠?

해당 생성자가 현재 같은 패키지 내 `Member` 생성자에서만 쓰이고 있던데,

```suggestion
    protected Profile() {
```

다른 생성자들과 접근 제어자 레벨을 맞추는 건 어떨까요? 🤔